doc/source/getting_started/troubleshooting.rst

```diff
@@ -0,0 +1,42 @@
+.. _troubleshooting:
+
+===============
+Troubleshooting
```

Comment:
There's already an FAQ document under user's guide.

Reply:
Putting this page with the same page of installation is extremely useful.

Reply:
I can delete the FAQ document then.